docs/packaging_policy.md

```diff
@@ -130,3 +130,14 @@ in this direction. However, there are a couple of options that have a special me
    ```
 
    ensuring that, when the option is active, the recipe ignores all the settings and only one package ID is generated.
+
+* `build_testing` should not be added, nor any other related unit test option. Options affect the package ID, therefore, testing should not be part of that.
+   Instead, use Conan config [skip_test](https://docs.conan.io/en/latest/reference/config_files/global_conf.html#tools-configurations) feature:
+
+   ```python
+   def _configure_cmake(self):
+      cmake = CMake(self)
+      cmake.definitions['BUILD_TESTING'] = bool(self.conf_info.get("tools.build:skip_test", check_type=bool))
```

Comment:
`conf_info`? Is it not `conf`?

Reply:
Right, I read again, and IHMO it's an ambiguous feature ... `self.conf` is available on the build context, but `conf_info` replaces `user_info` and `env_info` ...

https://docs.conan.io/en/latest/reference/config_files/global_conf.html#configuration-in-your-recipes
https://docs.conan.io/en/latest/reference/conanfile/attributes.html#conf
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#removed-self-user-info